sdk/core/core-client-rest/src/urlHelpers.ts

```diff
@@ -103,7 +103,7 @@ function buildRoutePath(
       value = encodeURIComponent(pathParam);
     }
 
-    routePath = routePath.replace(/{([^/]+)}/, value);
+    routePath = routePath.replace(/{([^/,]+)}/, value);
```

Comment:
are there other valid separators that we care about?

Reply:
I haven't found any other separators. However your comment got me thinking we could use a more robust regex like

`/\{\w+\}/` which doesn't really need it to care about separators